### PR TITLE
bpo-45056: Remove trailing unused constants from co_consts

### PR DESCRIPTION
**Pull request by methane**
https://bugs.python.org/issue45056

### CODE REVIEW

**Comment by pablogsal** [comment]
You need to regenerate `importlib` no?

**Comment by methane** [comment]
> You need to regenerate `importlib` no?

Yes. But it will cause conflict soon. So I want to delay `make regen` after code review.

File: Python/compile.c [hunk 1]
```diff
@@ -7868,6 +7871,9 @@ assemble(struct compiler *c, int addNone)
     if (duplicate_exits_without_lineno(c)) {
         return NULL;
```
Comment (gvanrossum) [inline Python/compile.c:7874]: Now, I am curious why this isn’t “go to error”.

**Review by gvanrossum — APPROVED** [review]

**Comment by gvanrossum** [comment]
What if unused constants appear in the middle? Can that happen?

**Comment by methane** [comment]
> What if unused constants appear in the middle? Can that happen?

Yes, there are some cases. This pull request doesn't remove them because I am focusing the regressin at Python 3.10.
I uploaded the dump of unused constants of `_pyio.py` file at the b.p.o. Please see the diff of them.
Python 3.10 appends unused `None` for many functions.

**Comment by ambv** [comment]
@methane, I'll deal with the backport.

**Pull request merged**